src/mockSystem.test.js

```diff
@@ -11,12 +12,13 @@ describe('mock system functions', () => {
   });
 
   it('gives simple 0s for readstat', done => {
+    expect.assertions(5);
     system.readstat().then(data => {
-      expect(data.utime).toBe(0);
-      expect(data.stime).toBe(0);
-      expect(data.cutime).toBe(0);
-      expect(data.cstime).toBe(0);
-      expect(data.rss).toBe(0);
+      _.chain(data)
```

Comment:
can we do this without _.chain? it's the tests but I find it easier to read code when we don't use _.chain